src/api/app/views/webui/project/subprojects.html.haml

```diff
@@ -10,12 +10,12 @@
     %thead
       %tr
         %th Parent project
-        %th Description
+        %th Title
     %tbody
       - @parentprojects.each do |project|
         %tr
-          %td= link_to project.name, :action => 'show', :project => project.name
-          %td= project.title
+          %td{:width => "50%"}= link_to project.name, :action => 'show', :project => project.name
```

Comment:
if this is about aligning the 3 tables you can also opt to use javascript to set the 2 other tables to the width of the largest one.